### PR TITLE
Chris Sach Green Grocers

### PR DESCRIPTION
**Pull request by ChrisJS90**

### CODE REVIEW

**Review by MaybeFreak — COMMENTED** [review]
Very nice code, lots of similarities to my own code so it was really easy to understand what was going on.  Maybe think about adding some comments for extra clarity.

File: index.js [hunk 1]
```diff
@@ -52,4 +52,117 @@ const state = {
...
+
+    const img = document.createElement('img');
+    img.src = 'assets/icons/' + item.id + '.svg';
+    img.alt = item.name;
+    
+    const addToCart = document.createElement('button');
+    addToCart.innerHTML = 'Add to cart';
+    addToCart.addEventListener('click', () => addItemToCart(item));
+
+    div.append(img);
```
Comment (JakeBurdass) [inline index.js:96]: Appending everything together makes it very readable

File: index.js [hunk 2]
```diff
@@ -52,4 +52,117 @@ const state = {
...
+}
+
+function UpdateTotal() {
+  let total = 0;
+  state.cart.forEach((item) => {
+    let cost = item.price * item.quantity;
+    total += cost;
+  })
+
+  total = total.toFixed(2);
```
Comment (JakeBurdass) [inline index.js:160]: I presume this fixes the total to two decimals? V nice

File: index.js [hunk 3]
```diff
@@ -52,4 +52,117 @@ const state = {
...
+
+  totalPrice.innerHTML = `£ ${total}`;
```
Comment (ayMissouri) [inline index.js:162]: I think this should probably be innerText rather than innerHTML

File: index.js [hunk 4]
```diff
@@ -52,4 +52,117 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+// Element selectors
```
Comment (ayMissouri) [inline index.js:57]: I like how you have split your code into different sections such as the selectors, rendering etc.

File: index.js [hunk 5]
```diff
@@ -52,4 +52,117 @@ const state = {
...
+      const addButt = document.createElement('button');
+      addButt.setAttribute('class', 'add-btn');
+      addButt.innerHTML = '+';
+      addButt.addEventListener('click', () => {
+        item.quantity++;
+        renderCart();
+        UpdateTotal();
+      })
+
+      li.append(img);
```
Comment (ayMissouri) [inline index.js:143]: appending everything together at the end makes it much more readable, i believe you could also append things on the same line for example: li.append(img, text, removeButt, span, addButt);

**Review by ayMissouri — COMMENTED** [review]
Very well done 👍

**Pull request closed without merging**